fili-core/src/main/java/com/yahoo/bard/webservice/util/CompletedFuture.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.util;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * A utility class to create futures which have already been completed
+ * and will return as soon as it is called.
+ *
+ */
+public class CompletedFuture {
+
+    /**
+     * Private constructor - all methods static.
+     */
+    private CompletedFuture() {
+        //Intentionally left blank.
+    }
+
+    /**
+     * Creates a completed {@link java.util.concurrent.Future} which will return this item.
+     *
+     * @param item  The item to return when called
+     * @param <E>  The type to be returned by this future
+     *
+     * @return a completed {@link CompletableFuture} which will successfully return an item.
+     */
+    public static <E> CompletableFuture<E> returning(E item) {
```

Comment:
I was using this in #430 but it's no longer needed and can be deleted if that makes sense.